doc/esmvalcore/preprocessor.rst

```diff
@@ -19,25 +48,18 @@ that some operations must be necessarily performed in a specific order. This is
 the case, for instance, for multi-model statistics, which required the model to
 be on a common grid and therefore has to be called after the regridding module.
 
-In this section, each of the preprocessor modules is described in detail
-following the default order in which they are applied:
-
-* `Variable derivation`_.
-* `CMOR check and dataset-specific fixes`_.
-* `Vertical interpolation`_.
-* `Land/Sea/Ice masking`_.
-* `Horizontal regridding`_.
-* `Masking of missing values`_.
-* `Multi-model statistics`_.
-* `Time operations`_.
-* `Area operations`_.
-* `Volume operations`_.
-* `Unit conversion`_.
+Features of the ESMValTool Climate data pre-processor are:
```

Comment:
Why did you remove the table of contents?
Please revert this change.
Also, the different sections were ordered as they are executed in the code by default, I would not change this.

Reply:
sorry, my bad - I re added it, you still think we should reorder the sections themselves or leave it for the user to click on the TOC link?

Reply:
Both. :smiley: 
I think the order should be consistent with the TOC.

Reply:
very very much copy pasting and deleting, I'd rather not do it since I am not good at copy-paste-delete :grin:

Reply:
I'll fix that while reading it again.

Reply:
you are a bonbon @mattiarighi :grin: